app/package/models.py

```diff
@@ -335,7 +340,8 @@ class MetaDataDB(db.Model):
     name = db.Column(db.TEXT, index=True)
     version = db.Column(db.TEXT, index=True, default='latest')
     descriptor = db.Column(db.JSON)
-    status = db.Column(db.TEXT, index=True, default='active')
+    status = db.Column(db.Enum(PackageStateEnum, native_enum=False),
+                       index=True, default=PackageStateEnum.active)
```

Comment:
Moved general issue on index performance to https://github.com/frictionlessdata/dpr-api/issues/183